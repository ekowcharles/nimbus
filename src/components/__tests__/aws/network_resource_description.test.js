import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import NetworkResourceDescription from '../../aws/network_resource_description'

describe('NetworkResourceDescription', () => {
  it('renders vpc or subnet description', () => {
    const wrapper = shallow(
      <NetworkResourceDescription
        cidr="10.0.1.0/24"
        hasIcon={false}
        hideCidr={false}
        hideId={false}
        id="subnet-44444444444444444"
      />
    )

    expect(toJson(wrapper)).toMatchSnapshot()
  })
})