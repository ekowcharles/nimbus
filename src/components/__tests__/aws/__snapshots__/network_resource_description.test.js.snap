// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NetworkResourceDescription renders vpc or subnet description 1`] = `
<div
  className="res-desc desc no-res-icon"
>
  subnet-44444444444444444 | 10.0.1.0/24
</div>
`;
